auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -341,6 +342,22 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
 
   /** Create a business record in Lear */
   private async createBusinessRecord (business: Business): Promise<string> {
+    let filingResponse = null
+    if ([LegalTypes.SP, LegalTypes.GP].includes(business.nameRequest.legalType as LegalTypes)) {
```

Comment:
Never a bad idea to optionally chain the deconstruction of an object in the event it's looking for the values before the object itself is available or something.

ie `business.nameRequest?.legalType`

Reply:
On the other hand, if you know for sure the object exists (eg, `business` as in Cam's example) then it doesn't need opt chaining. (Good example Cam.)

What I'm saying is we shouldn't use opt chaining on everything just because :)